patches/react-native-push-notification+3.0.2.patch

```diff
@@ -0,0 +1,53 @@
+patch-package
+--- a/node_modules/react-native-push-notification/android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java
```

Comment:
Questa patch non va bene, viene chiamato due volte onNotification quando si clicca su una notifica mentre l'app è in background.